`cleanLssl` in `gulp clean` fails

Not sure if this happens in `master`.

```sh
gulp clean
```

```
[14:45:16] Using gulpfile ~/Gulpfile.js
[14:45:16] Starting 'clean'...
[14:45:16] Starting 'cleanScripts'...
[14:45:16] Starting 'cleanLib'...
[14:45:16] Starting 'cleanDiagnostics'...
[14:45:16] Starting 'cleanTsc'...
[14:45:16] Starting 'cleanServices'...
[14:45:16] Starting 'cleanServer'...
[14:45:16] Starting 'cleanLssl'...
[14:45:16] Starting 'cleanTests'...
[14:45:16] Starting 'cleanRules'...
[14:45:16] Starting 'cleanCancellationToken'...
[14:45:16] Starting 'cleanTypingsInstaller'...
[14:45:16] Starting 'cleanWatchGuard'...
[14:45:16] Starting 'cleanTypesMap'...
[14:45:16] Starting 'cleanWebHost'...
[14:45:16] Starting 'cleanLoggedIO'...
[14:45:16] Starting 'cleanInstrumenter'...
[14:45:16] Starting 'cleanImportDefinitelyTypedTests'...
[14:45:16] Starting 'cleanReleaseTsc'...
[14:45:16] Finished 'cleanLoggedIO' after 21 ms
[14:45:16] Finished 'cleanLib' after 26 ms
[14:45:16] Finished 'cleanDiagnostics' after 28 ms
[14:45:16] Finished 'cleanServices' after 28 ms
[14:45:16] Finished 'cleanTypesMap' after 27 ms
[14:45:16] Finished 'cleanLssl' after 28 ms
error TS6053: File '/home/daniel/shared/TypeScript3/tests/webhost/webtsc.tsconfig.json' not found.
```

--- a/Gulpfile.js
+++ b/Gulpfile.js
@@ -454,16 +454,6 @@ task("baseline-accept").description = "Makes the most recent test results the ne
 task("baseline-accept-rwc", () => baselineAccept(localRwcBaseline, refRwcBaseline));
 task("baseline-accept-rwc").description = "Makes the most recent rwc test results the new baseline, overwriting the old baseline";
 
-// TODO(rbuckton): Determine if 'webhost' is still in use.
-const buildWebHost = () => buildProject("tests/webhost/webtsc.tsconfig.json");
-task("webhost", series(lkgPreBuild, buildWebHost));
-task("webhost").description = "Builds the tsc web host";
-
-const cleanWebHost = () => cleanProject("tests/webhost/webtsc.tsconfig.json");
-cleanTasks.push(cleanWebHost);
-task("clean-webhost", cleanWebHost);
-task("clean-webhost").description = "Cleans the outputs of the tsc web host";
-
 const buildLoggedIO = async () => {
     mkdirp.sync("built/local/temp");
     await exec(process.execPath, ["lib/tsc", "--types", "--target", "es5", "--lib", "es5", "--outdir", "built/local/temp", "src/harness/loggedIO.ts"]);

